dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1110,278 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: Union[Any, bytes, None] = None,
+            instance_id: Optional[str] = None,
+            workflow_options: Optional[Dict[str, str]] = dict()) -> StartWorkflowResponse:
```

Comment:
If you are assigning a default value of `dict()` anyway, then you don't need this to be `Optional` - only use `Optional` when you want to set it to `None` by default or you want to explicitly support `None`.